classes/Plane.php

```diff
@@ -3,10 +3,32 @@
 
 class Plane extends  Unit
 {
+
     public function __construct()
     {
         $this->life = 50;
         $this->attackPower = 15;
+        $this->attackType = "plane";
```

Comment:
Here I see a potential problem... You hardcoded the attackType with "plane" but what if is "Plane"?  And also we know that this is Plane class maybe you could determine wich class is  with php function get_class(). Take a look at it. every time you have to hardcode something there is a potential issue. What if you will chang class name from plane to FastPlane?